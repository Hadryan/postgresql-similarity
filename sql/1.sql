CREATE EXTENSION similarity;

SELECT similarity('Геометрия', 'Астрономия', 0);
