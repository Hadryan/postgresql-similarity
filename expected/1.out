    similarity     
-------------------
 0.421052631578947
(1 row)

